src/gui/embed.cpp

```diff
@@ -39,15 +42,70 @@ auto loadPixmap(const QString& name, int width, int height, const char* const* x
 	if (xpm) { return QPixmap{xpm}; }
 
 	const auto resourceName = QDir::isAbsolutePath(name) ? name : "artwork:" + name;
-	auto reader = QImageReader{resourceName};
-	if (width > 0 && height > 0) { reader.setScaledSize(QSize{width, height}); }
 
-	const auto pixmap = QPixmap::fromImageReader(&reader);
-	if (pixmap.isNull()) {
-		qWarning().nospace() << "Error loading icon pixmap " << name << ": " << reader.errorString();
-		return QPixmap{1, 1};
+	qreal devicePixelRatio = QGuiApplication::primaryScreen()->devicePixelRatio();
+
+	// Check if the resource is an SVG or a raster image
+	QImageReader reader(resourceName);
+	const QByteArray format = reader.format();
+
+	if (format.toLower() == "svg")
+	{
+		// Handle SVG with QSvgRenderer
+		QFile file(resourceName + ".svg");
```

Comment:
When I was investigating this, I had issues loading the resource too and I couldn't figure out why.  We should at least document this because for some reason `QImageReader`doesn't need this suffix.